### DISCUSSION TITLE
Creating and closing floating window.

### DISCUSSION THREAD

**Discussion by ardijanr** [hecrj/iced#685] [Q&A] [answered]
Hi, I am trying to make floating window on a tiling window manager, but looking around in the docs I cant seem to find any mention of it. It should just be a flag or something that needs to be set for it to recognize that it needs to be floating. I understand that this is the window managers job to handle, just exploring the possibility. 

Also, other than killing the process is there a way a command/function to close the window from iced?
----Not yet implemented, 
Issue: https://github.com/hecrj/iced/issues/226

works with panic!() but don't really want to panic....

Currently this works, nicer than panic:
std::process::exit()

**Comment by ardijanr** [comment, marked as answer]
Forcing window size seems to do the trick for floating window on tiling window managers. 

My solution:

> fn main() -> iced::Result {
>     let settings = Settings {
>             window: window::Settings {
>                 max_size: Some((400,450)),
>                 resizable: false,
>                 ..window::Settings::default()
>             },
>             ..Default::default()
>         };
>     ScreenMode::run(settings)
> }

**Comment by dhardy** [comment]
I believe this is a winit issue, e.g.: https://github.com/rust-windowing/winit/issues/862 https://github.com/rust-windowing/winit/issues/950

**Reply by ardijanr** [reply]
Yeah, there does not seem to be a standard for setting this, would be nice though. Though i think for popups most tiling window managers handle dialog boxes quite well. Its when you want your main window to be a floating window the issue arises.
